tasks/recover/print_info.yml

```diff
@@ -0,0 +1,17 @@
+- name: Print info about VMs that were registered
+  debug:
+    msg: "The following VMs were successfully registered: {{ succeed_vm_names }}"
+  when: succeed_vm_names | length > 0
+- name: Print info about VMs that were not registered
```

Comment:
Please repharse this to be compatible to the message:
   Print info about VMs which failed to be registered